app/konnect/gateway-manager/data-plane-nodes/index.md

```diff
@@ -45,7 +45,7 @@ Advanced setup:
 {{site.konnect_product_name}} supports using non-transparent forward proxies to connect your {{site.base_gateway}} data plane with the {{site.konnect_saas}} control plane. See the [Forward proxy connections](/gateway/latest/production/networking/cp-dp-proxy/) {{site.base_gateway}} documentation for more information.
 
 ## Data plane node dashboard
```

Comment:
"KIC data plane node dashboard" (line 52) is still an H3 while this is an h2, should make them the same heading level.